output/cmf.eslint.txt

```diff
@@ -8,12 +8,15 @@ The react/require-extension rule is deprecated. Please use the import/extensions
 /home/travis/build/Talend/ui/packages/cmf/src/Inject.component.js
   23:2  warning  Unexpected console statement  no-console
 
+/home/travis/build/Talend/ui/packages/cmf/src/onError.js
+  126:33  error  'addSensibleKeyRegexp' is not defined  no-undef
```

Comment:
to use a function addSensibleKeyRegexp not declared in https://github.com/Talend/ui/pull/2485/files#diff-fea8ec2c9e85e497f04b0115a332439eR126

Reply:
already fixed